### PR TITLE
clean up sample tests

### PR DESCRIPTION
**Pull request by ianwremmel**
- [x] make `setValue()` a wdio command
- [x] make `waitForSpecificText()` a wdio command
- [x] fix lint warnings
- [x] reset at start of tests
- [x] remove all direct uses of `waitFor()`
- [x] remove all direct uses of `assert.*()`
- [x] normalize `index.html` and `app.js`

### CODE REVIEW

File: packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/dial-and-reject.js [hunk 1]
```diff
@@ -13,12 +12,14 @@ describe('sample-browser-multi-party-call', () => {
     const browserSpock = browser.select('browserSpock');
     const browserMccoy = browser.select('browserMccoy');
 
-    before('reload browser', () => browser.refresh());
+    before('reload browser', () => {
+      browser.select('browserSpock').reload();
```
Comment (adamweeks) [inline packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/dial-and-reject.js]: You have the `const`'s up above, do you need to do the select over again? Also, `reload` spawns a whole new selenium test, did you mean to do a `refresh`?

File: packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/dial-before-connect.js [hunk 2]
```diff
@@ -13,12 +12,14 @@ describe('sample-browser-multi-party-call', () => {
     const browserSpock = browser.select('browserSpock');
     const browserMccoy = browser.select('browserMccoy');
 
-    before('reload browser', () => browser.refresh());
+    before('reload browser', () => {
+      browser.select('browserSpock').reload();
```
Comment (adamweeks) [inline packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/dial-before-connect.js]: Same thing here, `reload` vs `refresh`

File: wdio.helpers.d/assertions.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+const addCommand = require('./lib/add-command');
```
Comment (adamweeks) [inline wdio.helpers.d/assertions.js]: These helpers would be useful in the react repo too. Should we make them a package or put them in the web styleguide repo?

File: packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/dial-and-reject.js (hunk 1 above)
Comment (ianwremmel) [reply packages/node_modules/samples/browser-multi-party-call/test/wdio/spec/dial-and-reject.js]: it was mostly a matter of copying this block into each test. wasn't really paying attention to what was or was not already defined.

Yes, I'm intentionally restarting selenium. The tests were getting stuck in weird states; it about doubles the test duration, but they're way more stable.

File: wdio.helpers.d/assertions.js (hunk 3 above)
Comment (ianwremmel) [reply wdio.helpers.d/assertions.js]: probably, but we really need a template for new standalone projects with semantic-release fully automated.

Comment (ianwremmel) [reply wdio.helpers.d/assertions.js]: we could put them in the styleguide, but then we'd probably need to justify it as a "style rule" somehow, though there is something to be said for disallowing `browser.waitUntil` in spec files. That still runs into the issue of not having a fully automated release process.

**Pull request merged**